test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -116,10 +119,36 @@ public static Service createPravegaSegmentStoreService(final URI zkUri, final UR
                 return  new PravegaSegmentStoreDockerService(serviceId, zkUri, hdfsUri, contUri);
             case KUBERNETES:
             default:
-                return new PravegaSegmentStoreK8sService(serviceId, zkUri);
+                return new PravegaSegmentStoreK8sService(serviceId, zkUri, getPravegaProperties());
+        }
+    }
+
+    public static PravegaProperties getPravegaProperties() {
+        if (!SECURITY_ENABLED) {
+            return PravegaProperties.builder().build();
+        } else {
+            return PravegaProperties.builder().buildWithSecurity();
         }
     }
 
+    public static ClientConfig getClientConfig(URI controllerUri) {
+        if (!SECURITY_ENABLED) {
+            log.info("Generating config with auth disabled.");
+            return ClientConfig.builder().controllerURI(controllerUri).build();
+        } else {
+            log.info("Generating config with auth enabled.");
+            return ClientConfig.builder()
+                               // auth
+                               .credentials(new DefaultCredentials("1111_aaaa", "admin"))
+                               // tls
```

Comment:
Please remove commented out code. It adds additional cognitive burden on the reader.

Reply:
Intentionally left this in, as it needs to be added in near future.

Reply:
I'd rather that we remove unused code. You might want to store it in your local repository in some way.

Reply:
done